pages/Interaction/long-animation-frames

```diff
@@ -0,0 +1,47 @@
+# Long Animation Frames
+
+To determine when long animation frames (LoAF) happen, you can use [PerformanceObserver](https://developer.mozilla.org/docs/Web/API/PerformanceObserver) and register to observe entries of type `long-animation-frame`. This snippet from [@noamr](https://github.com/noamr) provides additional informations computed from the LoAF raw data.
```

Comment:
The URL it's broken

![image](https://github.com/nucliweb/webperf-snippets/assets/1307927/560be44a-15fd-43cf-a76d-1800b39dd441)